src/GraphQLinq.Client/GraphContext.cs

```diff
@@ -8,15 +10,31 @@ namespace GraphQLinq
 {
     public class GraphContext
     {
-        protected GraphContext(string baseUrl, string authorization)
+        public HttpClient HttpClient
         {
-            BaseUrl = baseUrl;
-            Authorization = authorization;
+            get;
+        }
+
+        protected GraphContext(HttpClient httpClient)
+        {
+            HttpClient = httpClient;
+            ContractResolver = new DefaultContractResolver();
+        }
+
+        protected GraphContext(string url, string authorization)
```

Comment:
I think it would also be great if we had another ctor accepting `HttpClientHandler` so that all HTTP aspects can be configured in one place. Maybe even better if it was integrated with `HttpClientFactory` too. As that requires .Net Standard 2.1 the project will need to target multiple frameworks I guess.

Reply:
Regarding HttpClientHandler. Most of the time it used like that:
```
HttpClientHandler handler = new HttpClientHandler();
HttpClient client = new HttpClient(handler);
```
If we accept the HttpClient, the developer has a way to configure it he wants(add a special handler for example) and then pass it to context. As a result, ctor with HTTP client is enough to cover the use cases.

Reply:
The same story with HttpClientFactory. It is initialized and can be reused multiple times outside the query context. As a result, its lifetime will be different. So, it is better to accept HTTP client because it shares the lifetime with the query context.

P.S. I need to account for HttpClient when disposing of the query context. I will take care of it in https://github.com/Giorgi/GraphQLinq/pull/26